src/dart_diagnostic_provider.ts

```diff
@@ -17,7 +17,7 @@ export class DartDiagnosticProvider {
 
 	private handleErrors(notification: as.AnalysisErrorsNotification) {
 		let errors = notification.errors;
-		if (!getConfig('showTodos'))
+		if (!getConfig<boolean>('showTodos'))
```

Comment:
I can't remember why, I tried to standardise on double quotes for strings. I seemed to be randomly mixing them and it kinda bugged me. Maybe I thought I'd be putting apostrophes in English strings more and I guess they're also slightly more distinguishable from template-strings (backticks).

You don't have to worry about changing this but I thought it worth mentioning (or maybe there's a reason I should switch!?)